.changes/v2.20.0/532-features.md

```diff
@@ -4,9 +4,9 @@
 * Added `NsxtEdgeGateway.GetUsedIpAddressSlice` method to fetch used IP addresses in a slice
   [GH-532]
 * Added `NsxtEdgeGateway.GetUnusedExternalIPAddresses` method that can help to find an unused
-  IP address in an Edge Gateway by given constraints [GH-532]
+  IP address in an Edge Gateway by given constraints [GH-532,GH-567]
 * Added `NsxtEdgeGateway.GetAllUnusedExternalIPAddresses` method that can return all unused IP
-  addresses in an Edge Gateway [GH-532]
+  addresses in an Edge Gateway [GH-532,GH-567]
```

Comment:
I have a doubt here, as I used a different syntax in my PR:
https://github.com/vmware/go-vcloud-director/pull/566
Although I tried to search and it seems both are used in changelog 😄 
Which one should we use?

Reply:
The answer is in `scripts/changelog-links.sh`

Such message
```
* Added `NsxtEdgeGateway.GetAllUnusedExternalIPAddresses` method that can return all unused IP addresses in an Edge Gateway [GH-532,GH-567]
```

Renders: 

* Added `NsxtEdgeGateway.GetAllUnusedExternalIPAddresses` method that can return all unused IP addresses in an Edge Gateway ([#532](https://github.com/vmware/go-vcloud-director/pull/532),[#567](https://github.com/vmware/go-vcloud-director/pull/567))


While such message

```
* Added support for Runtime Defined Entity Interfaces with client methods
  `VCDClient.CreateDefinedInterface`, `VCDClient.GetAllDefinedInterfaces`,
  `VCDClient.GetDefinedInterface`, `VCDClient.GetDefinedInterfaceById` and methods to manipulate
  them `DefinedInterface.Update`, `DefinedInterface.Delete` [GH-527], [GH-566]
```

Renders:
* Added support for Runtime Defined Entity Interfaces with client methods
  `VCDClient.CreateDefinedInterface`, `VCDClient.GetAllDefinedInterfaces`,
  `VCDClient.GetDefinedInterface`, `VCDClient.GetDefinedInterfaceById` and methods to manipulate
  them `DefinedInterface.Update`, `DefinedInterface.Delete` ([#527](https://github.com/vmware/go-vcloud-director/pull/527)], [[#566](https://github.com/vmware/go-vcloud-director/pull/566))

Reply:
Thanks! This clarifies it! Time to change #566 👍